samples/ingestion/ingestion-client/FetchTranscription/Program.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="Program.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscription
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.WorkerService;
+    using Microsoft.Azure.Functions.Worker;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    public static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var host = new HostBuilder()
+                .ConfigureFunctionsWorkerDefaults()
+                .ConfigureServices(s =>
```

Comment:
>ConfigureServices [](http://example.com/codeflow?start=17&length=17)

looking at the migration guide, it seems to be recommended to move all DI to Program.cs ConfigureServices()

[https://learn.microsoft.com/en-us/azure/azure-functions/migrate-dotnet-to-isolated-model?tabs=net8#programcs-file](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-dotnet-to-isolated-model?tabs=net8#programcs-file)


'The Program.cs file will replace any file that has the FunctionsStartup attribute, which is typically a Startup.cs file. In places where your FunctionsStartup code would reference IFunctionsHostBuilder.Services, you can instead add statements within the .ConfigureServices() method of the HostBuilder in your Program.cs.'



Let's maybe get rid of the Startup.cs in FetchTranscription function and move it to Program.cs?

Reply:
Great spot - I didn't even notice that Startup class!! :-)